src/main/java/mil/dds/anet/resources/ReportResource.java

```diff
@@ -135,6 +137,8 @@ public Report createReport(@GraphQLRootContext Map<String, Object> context,
       throw new WebApplicationException("Can only create Draft reports", Status.BAD_REQUEST);
     }
 
+    assertAllowedClassification(r.getClassification());
```

Comment:
There's now a static import for this, and a regular import for `ResourceUtils` (for the call to `ResourceUtils.checkAndFixNote`), which is a bit inconsistent. Pick either one (I have a slight preference for just a regular import of `ResourceUtils` in this case).